### PR TITLE
Map homepage

### PR DESCRIPTION
**Pull request by Franck59840**

### CODE REVIEW

**Review by Kengaroo — APPROVED** [review]

**Comment by Kengaroo** [comment]
Conflicting files
public/assets/css/header.css
src/View/Home/index.html.twig
src/View/Item/_header.html.twig
src/View/layout.html.twig 

Fais merge avec dev, stp

**Review by Kengaroo — APPROVED** [review]

File: database.sql [hunk 1]
```diff
@@ -1,63 +1,69 @@
...
+
+/*Table structure for table `announcement` */
+
+CREATE TABLE `announcement` (
+  `id` int(7) NOT NULL AUTO_INCREMENT,
+  `regionID` int(3) NOT NULL,
+  `title` char(25) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `message` varchar(500) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `address` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `authorID` int(7) NOT NULL,
```
Comment (Benja-san) [inline database.sql]: Look at the requested changes for column left, change author_id and region_id in sql file

**Review by Benja-san — COMMENTED** [review]
See the column review before repush on your branch

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -34,7 +25,6 @@
```
Comment (Benja-san) [inline src/View/layout.html.twig:25]: no content block left!

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/layout.html.twig (hunk 2 above)
Comment (Franck59840) [reply src/View/layout.html.twig:25]: it's normal because left is a other feature

Comment (Franck59840) [reply src/View/layout.html.twig:25]: it's normal , the block left is other feature

**Review by Benja-san — APPROVED** [review]

**Pull request merged**